Libraries/LibWeb/Editing/Internal/Algorithms.cpp

```diff
@@ -4388,7 +4388,7 @@ bool values_are_equivalent(FlyString const& command, Optional<String> a, Optiona
 }
 
 // https://w3c.github.io/editing/docs/execCommand/#loosely-equivalent-values
-bool values_are_loosely_equivalent(FlyString const& command, Optional<String> a, Optional<String> b)
+bool values_are_loosely_equivalent(FlyString const& command, Optional<Utf16View> a, Optional<Utf16View> b)
```

Comment:
It seems every caller to this function is casting from `Optional<Utf16String>` to `Optional<Utf16View>` at the call site. Maybe this should just take the former?

BTW I think all the `Optional<String>` / `Optional<Utf16String>` could be `Optional<String const&>` and `Optional<Utf16String const&>` to avoid ref-count churn.

Reply:
> BTW I think all the Optional<String> / Optional<Utf16String> could be Optional<String const&> and Optional<Utf16String const&> to avoid ref-count churn.

Agreed, done!